ecs-deploy

```diff
@@ -323,7 +323,7 @@ function createNewTaskDefJson() {
 
     # Updated jq filters for AWS Fargate
     REQUIRES_COMPATIBILITIES=$(echo "${DEF}" | jq -r '. | select(.requiresCompatibilities != null) | .requiresCompatibilities[]')
-    if [[ "${REQUIRES_COMPATIBILITIES}" == 'FARGATE' ]]; then
+    if `echo ${REQUIRES_COMPATIBILITIES[@]} | grep -q "FARGATE"`; then
```

Comment:
this could be written as `if [[ "${REQUIRES_COMPATIBILITIES[@]}" =~ "FARGATE" ]]; then`